wire-ios-sync-engine/Source/Use cases/CreateTeamOneOnOneConversationUseCase.swift

```diff
@@ -112,11 +114,15 @@ struct CreateTeamOneOnOneConversationUseCase: CreateTeamOneOnOneConversationUseC
             throw Error.mlsMigratorNotFound
         }
 
-        let groupID: MLSGroupID
+        let groupID = try await syncMLSConversationFromBackend(
+            userID: userID,
+            in: context
+        )
```

Comment:
Does it make sense that we sync from backend in case we `createMLSConversation`?
If not how to retrieve the groupID, that is needed for `migrateToMLS`?